### ISSUE TITLE
Own City Info panel collapses when hitting the copy button and copies the word Copy

### ISSUE THREAD

**Issue by kronikpillow** [FoE-Info/FoE-Info-Extension#37]
in the city info panel 
![city info](https://i.imgur.com/5VMq38X.png)
when you hit the copy button, the panel collapses ... this is not desired behavior, the panel should only collapse when you double click on the top of the panel, otherwise it should stay expanded when pressing the copy button

also, hitting the Copy button for some reason copies the word Copy and gives the following paste:

```
CopyEN16 KronikPillow
Daily: 598FP, 17M Coins
Goods: TF:335 TE:205 PME:50
Arc Bonus: 94%
Château Bonus: 390% / 25 Goods
Guild Goods: 2205
Guild Power: 2104
Army Units: 80
Attackers: 1043% Att, 565% Def
Defenders: 102% Att, 141% Def
Available FP: 215708
```
the word Copy, should not actually be copied

**Comment by kronikpillow** [comment]
@avengerrevolt in PR #44 panel now doesn't collapse, but copy output still copies word Copy
output of copy:
```
CopyEN7 KronikPillow
Daily: 798FP, 39M Coins
Goods: SAJM:388 SAV:252 SAM:144
Arc Bonus: 99.7%
Château Bonus: 590% / 35 Goods
Guild Goods: 2910
Guild Power: 1514
Army Units: 148
Attackers: 2007% Att, 1155% Def
Defenders: 143% Att, 188% Def
Available FP: 3555
```

it should be
```
EN7 KronikPillow
Daily: 798FP, 39M Coins
Goods: SAJM:388 SAV:252 SAM:144
Arc Bonus: 99.7%
Château Bonus: 590% / 35 Goods
Guild Goods: 2910
Guild Power: 1514
Army Units: 148
Attackers: 2007% Att, 1155% Def
Defenders: 143% Att, 188% Def
Available FP: 3555
```

**Comment by kronikpillow** [comment]
all fixed in PR #47
